tests/contrib/ray/test_ray.py

```diff
@@ -215,6 +215,17 @@ def add_one(x):
         assert running == [], f"Expected no running tasks, got {len(running)}"
         assert ray.get(done) == [43], f"Expected done to be [43], got {done}"
 
+    @pytest.mark.snapshot(token="tests.contrib.ray.test_ray.test_simple_get", ignores=RAY_SNAPSHOT_IGNORES)
+    def test_simple_get(self):
+        with override_config("ray", dict(trace_core_api=True)):
+
+            @ray.remote
+            def add_one(x):
+                return x + 1
+
+            results = ray.get([add_one.remote(x) for x in range(4)])
+            assert results == [1, 2, 3, 4], f"Expected [1, 2, 3, 4], got {results}"
```

Comment:
Just to be sure we are not OOMed again, I think you can do range(1) and change the expected results as well. What we are really testing here is get.